doc/Tex/RAM_SCB_run.tex

```diff
@@ -139,115 +82,48 @@
+
+For an example of restarting a simulation, run Test 2 and inspect the {\tt PARAM.in} files as well as the contents of {\tt restartIN} and {\tt restartOUT}.
 
 \section{Output}
-RAM-SCB, by default, has a rich output set.  This can be expanded by activating
-other output file types via the {\tt PARAM} interface, be sure to review
-the commands listed in Chapter \ref{chp:param}.  Table \ref{tab:out}
-summarizes the output that can be generated by RAM-SCB.  Visualization of
-these files is covered in Chapter \ref{chp:viz}.
+RAM-SCB, by default, has a rich output set. This can be expanded by activating other output file types via the {\tt PARAM} interface, be sure to review the commands listed in Chapter \ref{chp:param}. Table \ref{tab:out} summarizes the output that can be generated by RAM-SCB. Visualization of these files is covered in Chapter \ref{chp:viz}.
 
 
 \begin{table}[ht]
-  \centering
-  \begin{tabular}{l l l l l}
-  \hline\hline
-   Type & Extension &  Format & Default? & Contents\\
-  \hline
-  Log file & \verb*q*.logq & ASCII & Yes & Dst, integrated values\\
-  Pressure files & \verb*l*.inl & ASCII & Yes & $\perp$ and $\parallel$ partial pressures\\
-  Trapped files & \verb*l*.tl & ASCII & Yes & Averaged trapped fluxes\\
-  E-Field files & \verb*l*.inl & ASCII & Yes & Equatorial electric potential\\
-  Boundary files & \verb*l*.datl & ASCII & Yes & Fluxes at the outer boundary\\
-  Full flux files & \verb*l*.ncl & NetCDF & No & Full equatorial flux information\\
-  3D flux files & \verb*l*.ncl & NetCDF & No & Fluxes in the full 3D domain\\
-  Virtual Satellites &\verb*l*.ncl & NetCDF & No & Satellite specific values\\
-  Field integral files &\verb*l*.datl & ASCII & Yes & $h$, $I$ geometric integrals\\
-  Magnetic field file &\verb*l*.ncl & NetCDF & No & Full 3D SCB field\\
-  Potential file &\verb*l*.ncl & NetCDF & No & Ionospheric and equatorial electric potential\\
-  3D pressure file&\verb*l*.ncl & NetCDF & No & Full 3D anisotropic pressure\\ 
-  \end{tabular}
+ \centering
+ \begin{tabular}{l l l l l}
+ \hline\hline
+  Type & Extension & Format & Default? & Contents\\
+ \hline
+ Log file & \verb*q*.logq & ASCII & Yes & Dst, integrated values\\
+ Pressure files & \verb*l*.inl & ASCII & Yes & $\perp$ and $\parallel$ partial pressures\\
+ Trapped files & \verb*l*.tl & ASCII & No & Averaged trapped fluxes\\
+ E-Field files & \verb*l*.inl & ASCII & No & Equatorial electric potential\\
+ Boundary files & \verb*l*.datl & ASCII & No & Fluxes at the outer boundary\\
+ Full flux files & \verb*l*.ncl & NetCDF & Yes & Full equatorial flux information\\

```

Comment:
If these are equatorial only, why are they called "full"? Okay, I know the answer to that, but it's still a little misleading... Is there any reason we can't change this to be "Equatorial Flux Files" and "3D Flux Files"?

Reply:
No reason I can think of. Actually, most of those output files don't even exist any more. They were either changed in format, or were merged with another file. There have also been several additional file outputs added. So that whole section needs to get redone